src/network/tcpsocket.c

```diff
@@ -228,16 +225,19 @@ void send_tcp(char *msg) {
     print_tcp_array();
     if (network_config.use_symm_enc) {
         int length_enc;
-        size_t msglen = strlen(msg);
-        char *enc = gcrypt_encrypt_msg(msg, msglen + 1, &length_enc);
+        size_t msglen = strlen(msg)+1;
```

Comment:
Same question here. Why only doing this if encryption is enabled?

Reply:
fixed